cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
Is this file necessary?

Reply:
This file was added in migration to v10, I removed it